fdbserver/storageserver.actor.cpp

```diff
@@ -799,6 +808,16 @@ struct StorageServer {
 		// Bytes of the mutations that have been added to the memory of the storage server. When the data is durable
 		// and cleared from the memory, we do not subtract it but add it to bytesDurable.
 		Counter bytesInput;
+		// Bytes pulled from TLogs, it counts the size of the key value pairs, without any overhead.
+		Counter logicalBytesInput;
+		// Bytes pulled from TLogs for moving-in shards.
+		Counter logicalBytesMoveInOverhead;
+		// Bytes committed to the underlying storage engine by SS, it counts the size of key value pairs.
+		Counter kvBytesCommit;
+		// Count of clearRange operatons to the storage engine.
+		Counter kvClearRange;
```

Comment:
Does kvClearRange only account for operation in [\x00, \xff) space?

Reply:
It counts all clearRange operations, added comments.